scripts/reborn_webui_v2_live_qa/test_run_live_qa.py

```
@@ -1254,6 +1254,49 @@ async def fake_sleep(_seconds):
+            "reason": "The response does not complete the task.",
+            "inference_usage": {
+                "source": "semantic_judge",
+                "model": "deepseek-ai/DeepSeek-V4-Flash",
+                "input_tokens": 10,
+                "output_tokens": 5,
+                "cache_read_input_tokens": 0,
+                "cache_creation_input_tokens": 0,
+            },
+        }
+
+        async def fake_judge(**_kwargs):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `fake_judge`

(ANN202)